demo_data/db.sql

```diff
@@ -41,14 +41,14 @@ DROP TABLE IF EXISTS %PREFIX%_sessions             CASCADE;
 DROP TABLE IF EXISTS %PREFIX%_texts                CASCADE;
 DROP TABLE IF EXISTS %PREFIX%_user_relations       CASCADE;
 DROP TABLE IF EXISTS %PREFIX%_user_relation_types  CASCADE;
-DROP TABLE IF EXISTS %PREFIX%_user_log             CASCADE;
```

Comment:
We shouldn't delete this line now because we want to simulate the update process from user_log to log_changes

Reply:
you are right, that was one of the changes I had to revert...